components/uuf-core/src/main/java/org/wso2/carbon/uuf/internal/deployment/AppCreator.java

```diff
@@ -131,9 +132,15 @@ public App createApp(AppReference appReference, String contextPath) {
                         InMemorySessionManagerFactory.class.getName(), this.getClass().getClassLoader()));
         SessionManager sessionManager = sessionManagerFactory.getSessionManager(appName, configuration);
 
+        // Get Authorizer.
+        Authorizer authorizer = configuration.getAuthorizer()
+                .map(authorizerClass -> pluginProvider.getPluginInstance(Authorizer.class, authorizerClass,
+                        this.getClass().getClassLoader()))
+                .orElse(null);
```

Comment:
if authorizer is null, then we should put a warn log.
`LOGGER.warn("No authorizer is configured for '{}' app.", appName);`